backend/models/distance.py

```diff
@@ -151,25 +154,43 @@ def closest_with_early_exit(self, origin, point_list, exit_distance):
         )
 
         if min_haversine_distance <= exit_distance:
+            if self.mode == 'time':
+                return min_haversine_distance / 104607.0 * 60.0, point_list[min_haversine_idx]
```

Comment:
I think this is assuming a certain driving speed. Could be good to add a comment saying what that is in miles per hour (or kmph).